.github/workflows/yocto-build-deploy.yml

```
@@ -949,6 +948,7 @@ jobs:
     # we should consider using GitHub hosted runners for the testbot workers.
     runs-on: ${{ matrix.runs_on || fromJSON('["self-hosted", "X64", "kvm"]') }}
     environment: ${{ matrix.environment }}
+    if: ${{ github.event_name == 'pull_request' ||  github.event_name == 'pull_request_target'}}
```

```suggestion
    if: github.event_name != 'push'
```
We need to allow tests to run on workflow dispatch. The only time we don't want tests is on push.